### ISSUE TITLE
ES6 beta - problems Safari

### ISSUE THREAD

**Issue by rkuhl** [katspaugh/wavesurfer.js#1099]
Great library!  
Using the ES6 beta ver `2.0.0-beta01`  
Noticed some problems on Safari v 10.0.2  
```javascript  
import WaveSurfer from "wavesurfer.js"
let wavesurfer = WaveSurfer.create( {container: "#waveform"} )
wavesurfer.on('ready', function () {
    wavesurfer.play();
});
wavesurfer.load("sample.mp3");
```
On Safari I get an error:  
_IndexSizeError (DOM Exception 1): The index is not in the allowed range._   
Everything works just fine on Chrome / Firefox.  
Thanks for your assistance!

**Comment by entonbiba** [comment]
#1102

**Comment by mspae** [comment]
Should be fixed once #1102 is released.
